src/plugins/libfabric/libfabric_backend.cpp

```
@@ -225,6 +249,83 @@ nixlLibfabricEngine::vramFiniCtx() {
+    pci_bus_id = "";
+    if (is_dev) {
+        char buf[32];
+        if (hipDeviceGetPCIBusId(buf, sizeof(buf), dev_id) == hipSuccess) {
+            pci_bus_id = std::string(buf);
+        }
+    }
+    return 0;
+}
+
+int
+nixlLibfabricRocmCtx::rocmUpdateCtxPtr(void *address, int expected_dev, bool &was_updated) {
```

Cppcheck (2.20.0)

[style] The function 'rocmUpdateCtxPtr' is never used.

(unusedFunction)